bazel/canbench.sh

```diff
@@ -43,8 +43,14 @@ elif [ "$1" = "--update" ]; then
         cp "${RUNFILES}/canbench_results.yml" "${REPO_RESULTS_PATH}"
     fi
 else
+    if [[ -z "${NOISE_THRESHOLD}" ]]; then
+        NOISE_THRESHOLD_ARG=""
+    else
+        NOISE_THRESHOLD_ARG="--noise-threshold ${NOISE_THRESHOLD}"
+    fi
```

Comment:
This would be simpler:
```suggestion
    NOISE_THRESHOLD_ARG="${NOISE_THRESHOLD:+--noise-threshold ${NOISE_THRESHOLD}}"
```

Reply:
Thanks! Done.